.github/bin/label_abandoned_prs.py

```diff
@@ -0,0 +1,42 @@
+import json
+import subprocess
+import datetime
+from datetime import timedelta
+
+pr_json = "gh pr list --json updatedAt,labels,number,comments"
+string = subprocess.check_output(pr_json, shell=True, text=True)
+output = json.loads(string)
+
+for x in output:
+    already_warned = False
+    string = x['updatedAt']
+    for comment in x['comments']:
+        if comment['body'] == 'This\
+                         PR has been inactive (no commits and no review comments)\
+                         for 12 days. If there is no new activity in the next 48 hours,\
+                         this PR will be labeled as Abandoned PR - Needs New Owner and\
+                         either another developer will finish the PR or it will be closed.':
+            already_warned = True
+    json_time = datetime.datetime.fromisoformat(string.replace('Z', '+00:00'))
+    eastern = datetime.timezone(datetime.timedelta(hours=-5))
+    et_time_update = json_time.astimezone(eastern)
+    today = datetime.datetime.now()
+    two_weeks = timedelta(weeks=2)
+    twelve_days = timedelta(days=12)
+    two_days = timedelta(days=2)
+    et_today = today.astimezone(eastern)
+    tdiff = et_today - et_time_update
```

Comment:
Do we need to use a timezone here since all we care about is difference?

Reply:
if you don't set both times into the same timezone it causes an error, so I figured it would be easiest to convert both into eastern time